lib/rules/selector-not-notation/index.mjs

```diff
@@ -114,8 +116,8 @@ const rule = (primary) => {
 					node: ruleNode,
 					result,
 					ruleName,
-					line: range.start.line,
-					...range,
+					index,
+					endIndex,
```

Comment:
Using `index` and `endIndex` is the only way to have the correct outcome within Stylelint `16.x`.

Ideally we would use `start/end` because that is slightly more performant.

But this isn't compatible with the planned deprecation of `line` as an optional argument.

Reply:
IIRC `getRange` was necessary because, _back then_, the clones—l169—were lacking some data. If the tests pass all should be good.